noodles-editor/public/noodles/california-earthquakes/README.md

```diff
@@ -0,0 +1,40 @@
+# California Earthquakes
+
+## Overview
+This example visualizes earthquake data for California, where the size of each point represents earthquake magnitude and colors represent intensity using a color ramp.
+
+## What It Demonstrates
+- **Data-driven sizing**: Circle radius scales with earthquake magnitude using square root for better visual perception
+- **Color ramps**: Using `ColorRampOp` to map magnitude values to a color gradient
+- **Value mapping**: `MapRangeOp` normalizes magnitude values for color mapping
+- **Accessor expressions**: JavaScript expressions to transform data values
+
+## Key Techniques
+- **Data source**: `FileOp` loads earthquake data from CSV
+- **Position accessor**: Extracts `[Longitude, Latitude]` from data
```

Comment:
Let's assume someone would copy/paste this into an accessor.. would it be "d.Longitude"?